client/src/plugins/usage-statistics/event-handlers/__tests__/DeploymentEventHandlerSpec.js

```diff
@@ -814,6 +822,118 @@ describe('<DeploymentEventHandler>', () => {
     });
 
 
+    describe('subprocess planes', () => {
+
+      describe('bpmn', () => {
```

Comment:
Just thinking: should we already enable this for type `'cloud-bpmn'` as well already?

Reason:
* Maybe this would not break anything and just work out of the box
* This would mitigate the risk, that we forget about metrics, once we enable the feature for cloud

Reply:
It should already be enabled for Cloud bpmn files (at least I did not explicitly disable it)

I'll check it out and add test cases for this